include/irr/asset/IMeshBuffer.h

```diff
@@ -63,6 +64,9 @@ class IMeshBuffer : public virtual core::IReferenceCounted
 
     //debug
     core::CLeakDebugger* leakDebugger;
+
+    uint8_t m_pushConstantsData[MAX_PUSH_CONSTANT_BYTESIZE];
```

Comment:
alignas(64) and switch places with other members (so we're not wasting lot of space on padding)

Reply:
resolved